lua/terrortown/menus/gamemode/thirdparty/vfire.lua

```diff
@@ -0,0 +1,15 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 99
+CLGAMEMODESUBMENU.title = "submenu_thirdparty_vfire_title"
+
+function CLGAMEMODESUBMENU:Populate(parent)
+	local form2 = vgui.CreateTTT2Form(parent, "header_thirdparty_vfire")
+	form2:MakeHelp({
+		label = "help_thirdparty_vfire"
+	})
+end
```

Comment:
We can do so but we add a whole menu just for saying jeah this random addon here is supported by TTT2. I don't think that is really useful. Also in my opinion i would also not include the vFire convars in TTT2, this just adds more dependency on another addon that we need to maintain the convars for and it also sets a precedent, for the future on how we add convars for other addons, which is not good:
Currently we only add our own convars to the menu and allow addons to provide their own, and that is fine. But i would not want to start adding convars for external addons, which causes a confusion on who is responsible for adding and maintaining those. If those would be specific convars for controlling the type of interaction between our addon and an external addon, which is implemented on our side, sure thing, but otherwise we should not do that.

Reply:
That is a fair point and it would mean that this menu should be completely removed. How do we communicate to players/admins that we have builtin vFire support though?

Reply:
Don't know, write it in the changelog and the workshop page or whatever, but we are not communicating many compatibilites that we have i think.
We currently have no place / way to do so, which we can add but i would not do so in this PR.